Applications/MainApplication/Gui/MainWindow.cpp

```diff
@@ -4,7 +4,7 @@
 #include <QFileDialog>
 #include <QToolButton>
 
-#include <Core/Utils/File/OBJFileManager.hpp>
+#include <Core/File/deprecated/OBJFileManager.hpp>

```

Comment:
The best way to do it is to file an issue so that we don't forget to put it back on once this pr is merged